app/WebParsing/UtsgJsonParser.hs

```diff
@@ -0,0 +1,80 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+{-
+module WebParsing.UtsgJsonParser
+     (getJSON) where
+-}
+
+import Data.Aeson
+import Data.List
+import Database.Tables
+import WebParsing.PrerequisiteParsing
+import Database.Persist.Sqlite (runSqlite, insertMany_)
+import qualified Data.HashMap.Lazy as M
+import qualified Data.Text as T
+import Config (databasePath)
+import Network.HTTP.Conduit (simpleHttp)
+
+{- URL to UofT courses (stored as JSON string) -}
```

Comment:
Our documentation style isn't great, but please use the Haddock syntax `-- | My comment` instead.